addons/hr/models/hr.py

```diff
@@ -107,6 +105,9 @@ def _default_image(self):
         image_path = get_module_resource('hr', 'static/src/img', 'default_image.png')
         return tools.image_resize_image_big(open(image_path, 'rb').read().encode('base64'))
 
+    resource_id = fields.Many2one(
+        'resource.resource', string='Resource',
+        auto_join=True, delegate=True, ondelete='cascade', required=True)
```

Comment:
For the sake of readability, I would put the parameter `delegate` first (this is part of the model definition), and other parameters like `auto_join` or `index` last (this is an implementation option).
```python
    resource_id = fields.Many2one(
        'resource.resource', string='Resource',
        delegate=True, ondelete='cascade', required=True, auto_join=True)
```